src/templates/CopyPage.js

```diff
@@ -1,36 +1,47 @@
 import React from 'react';
 import { graphql } from 'gatsby';
 
-import { Layout, Link } from '@components';
+import { FocusBoundary, Layout, Link } from '@components';
 import { getAnchor } from '@utils/anchor';
 
 export default function Transcript({ data }) {
   const { html, frontmatter, headings } = data.markdownRemark;
 
   return (
     <Layout title={frontmatter.title} sidebar={frontmatter.sidebar}>
-      {(toggleSidebar) => (
+      {(setSidebar) => (
         <>
           <h1>{frontmatter.title}</h1>
+          {frontmatter.sidebar ? (
+            <FocusBoundary onChange={setSidebar}>
+              <nav>
+                <ol>
+                  {headings
+                    .filter((heading) => heading.depth < 3)
+                    .map(({ value }) => (
+                      <li key={value}>
+                        <Link
+                          to={getAnchor(value)}
+                          onClick={() => {
+                            setSidebar(false);
+                            document
+                              .getElementById(getAnchor(value).replace('#', ''))
+                              .querySelector('a')
+                              .focus();
```

Comment:
This skips the focus to the link next to the intended piece of content. In an ideal world we'd also have a way to skip the nav back to the sidebar instantly, but that's a lesser problem than leaving the focus in the sidebar when viewing content would've been!